encoding/rs/frame.go

```diff
@@ -33,3 +37,106 @@ func Decode(b []byte) (Frame, error) {
 	}
 	return f, nil
 }
+
+// GnarkEncodeFrames serializes a slice of frames into a byte slice.
+func GnarkEncodeFrames(frames []*Frame) ([]byte, error) {
+
+	// Serialization format:
```

Comment:
Should we move this to above the function so it shows up in the go docs?

Reply:
Done.

```
// GnarkEncodeFrames serializes a slice of frames into a byte slice.
//
// Serialization format:
// [number of frames: 4 byte uint32]
// [size of frame 1: 4 byte uint32][frame 1]
// [size of frame 2: 4 byte uint32][frame 2]
// ...
// [size of frame n: 4 byte uint32][frame n]
//
// Where relevant, big endian encoding is used.
func GnarkEncodeFrames(frames []*Frame) ([]byte, error) {
```